mongomock/aggregate.py

```diff
@@ -626,6 +627,61 @@ def _handle_conditional_operator(self, operator, values):
             'aggregation pipeline, it is currently not implemented '
             ' in Mongomock.' % operator)
 
+    def _handle_control_flow_operator(self, operator, values):
+        if operator == '$switch':
+            if not isinstance(values, dict):
+                raise OperationFailure(
+                    '$switch requires an object as an argument, '
+                    'found: %s' % type(values)
+                )
+
+            branches = values.get('branches', [])
+            if not isinstance(branches, list):
```

Comment:
Please also accept tuples.

Reply:
Support for the `tuple` added. Validation of it has been added to the tests.